Includes/networking.cpp

```diff
@@ -70,7 +71,12 @@ int setupNetwork(void* DHCP) {
   packet_timer(NULL);
 
   if (dhcp) {
+    time_t start = time(NULL);
     while (dhcp_supplied_address(g_pnetif) == 0) {
+      if((time(NULL) - start) > 7) {
```

Comment:
Why 7 seconds?

Reply:
I believe it's a reasonable time frame

Reply:
Doesn't 10 seconds sound better than an odd 7? :P

Reply:
I'm thinking 5, so this is probably a good compromise. 👍

Reply:
FWIW, ISC dhclient starts off with a 5s timeout.